### DISCUSSION TITLE
Lost all settings from a power outage

### DISCUSSION THREAD

**Discussion by GStudios1** [OrcaSlicer/OrcaSlicer#6918] [General]
Yesterday I was working on some drawings with Orca open (latest version) and the power started flickering and finally the computer shut down. I went back into Orca and it acted like I never  used it, it wanted me to set up everything from scratch, all my settings, history everything was gone, I tried loading the Qidi printers into it and those options were gone. I re-downloaded Orca and still now Qidi printers to choose from, really strange. Anyone else have this happen? is there a way to get the settings back? spent a lot of time fine tunning it for 2 different printes.

**Comment by localgamerz** [comment]
just had this happen as well, upsetting for sure
